fairlearn/reductions/_grid_search/grid_search.py

```diff
@@ -142,23 +82,12 @@ def __init__(self,
         self.grid_limit = float(grid_limit)
         self.grid = grid
 
-        self._all_results = []
-        self._best_result = None
-
-    @property
-    def all_results(self):
-        """Return a list of :class:`GridSearchResult` from each point in the grid."""
-        return self._all_results
-
-    @property
-    def best_result(self):
-        """Return the best result found from the grid search.
-
-        The predictor contained in this instance of
-        :class:`GridSearchResult` is used in calls to
-        :code:`predict` and :code:`predict_proba`.
-        """
-        return self._best_result
+        self._best_grid_index = None
+        self._predictors = []
+        self._lambda_vecs = pd.DataFrame()
+        self._objectives = []
+        self._gammas = pd.DataFrame()
+        self._oracle_calls_execution_time = []
```

Comment:
i think i prefer `oracle_execution_times

Reply:
works for me!